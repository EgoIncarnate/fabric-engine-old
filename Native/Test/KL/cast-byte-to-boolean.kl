operator entry()
{
  report Boolean(Byte(0));
  report Boolean(Byte(1));
  report Boolean(Byte(255));
}
