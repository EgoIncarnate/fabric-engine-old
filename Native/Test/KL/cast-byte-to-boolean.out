GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          CastNode( Boolean )
            CastNode( Byte )
              ConstSize( 0 )
        Report
          CastNode( Boolean )
            CastNode( Byte )
              ConstSize( 1 )
        Report
          CastNode( Boolean )
            CastNode( Byte )
              ConstSize( 255 )
false
true
true
